static/js/emoji_picker.js

```diff
@@ -216,15 +216,15 @@ function filter_emojis() {
             const emojis = category.emojis;
 
             for (const emoji_dict of emojis) {
-                _.any(emoji_dict.aliases, function (alias) {
+                for (const alias of emoji_dict.aliases) {
```

Comment:
It's worth a one-line comment here that the intention of this function is to just find one match per alias.  At least I think that's what it's doing.

Unfortunately, the original commit doesn't add much context here, as it doesn't have unit tests and the commit message doesn't add much.  5662726a228a7f1d9bce34529c50c14d2c78fc71

It does reference #6250, if that provides any insight.

Reply:
So #6250 actually does explain the intention of the code here.  My suggested comment would be something like

~~~
// We only need the first alias for any given emoji in our
// search results, so we loop through aliases until we find
// the first match then break out of the loop.
~~~

Reply:
Added a terser comment explaining the intention directly on the `break` line; the mechanics are apparent by inspection.